### PR TITLE
861m8y5fg-4: catalog cards amount

### PR DESCRIPTION
**Pull request by nastyakuzusha**

### CODE REVIEW

File: front/src/components/page-configurator/engine/Engine.tsx [hunk 1]
```diff
@@ -11,12 +11,12 @@ interface EngineProps {
...
     const [ratio, setRatio] = useState<number | null>(null);
-    const { breakpoints } = useMantineTheme();
+    const { breakpoints } = useMantineTheme().other;
 
     useEffect(() => {
-        setRatio(window.innerWidth < breakpoints.xl ?
-            window.innerWidth * 0.4 / data.width :
-            1400 * 0.4 / data.width);
+        setRatio(window.innerWidth < breakpoints.laptopXXL ?
+            window.innerWidth * 0.45 / data.width :
```
Comment (AbrosimovVladislav) [inline front/src/components/page-configurator/engine/Engine.tsx]: 0.45 и 0.5 should be as params in some params files with meaningful names

File: front/src/components/page-configurator/engine/EngineStyles.tsx [hunk 2]
```diff
@@ -6,17 +6,20 @@ interface engineStylesProps {
 
 export const useStyles = createStyles((theme, {width}: engineStylesProps) => ({
     canvas: {
-        height: "300px",
+        height: [`${width / 1.3}px`],
```
Comment (AbrosimovVladislav) [inline front/src/components/page-configurator/engine/EngineStyles.tsx]: 1.3 should be as params in some params files with meaningful names

File: front/src/components/page-configurator/engine/EngineStyles.tsx [hunk 3]
```diff
@@ -6,17 +6,20 @@ interface engineStylesProps {
...
         width: "100%",
         boxShadow: theme.shadows.xl,
         borderRadius: theme.other.spacing.xxs,
         display: "grid",
         placeItems: "center",
+        [`@media (min-width: ${theme.other.breakpoints.laptopM}px)`]: {
+            height: [`${width / 1.7}px`],
```
Comment (AbrosimovVladislav) [inline front/src/components/page-configurator/engine/EngineStyles.tsx]: 1.7 should be as params in some params files with meaningful names

**Comment by AbrosimovVladislav** [comment]
Should it be 5 cards here? Which number we discussed previously?
Снимок экрана 2023-02-07 в 12 54 43

**Comment by AbrosimovVladislav** [comment]
Should it be smaller abount here?
Снимок экрана 2023-02-07 в 12 55 48

**Comment by nastyakuzusha** [comment]
> Should it be 5 cards here? Which number we discussed previously? Снимок экрана 2023-02-07 в 12 54 43
we didnt discuss numbers

**Comment by nastyakuzusha** [comment]
> Should it be smaller abount here? Снимок экрана 2023-02-07 в 12 55 48

how much?

**Comment by AbrosimovVladislav** [comment]
> > Should it be 5 cards here? Which number we discussed previously? Снимок экрана 2023-02-07 в 12 54 43
> > we didnt discuss numbers

we did
Gun Part List Cards amount (2-3-4-5-6?)REVIEW
Check with 5, make choice what is better 4 or 5 by yourself

**Comment by AbrosimovVladislav** [comment]
> > Should it be smaller abount here? Снимок экрана 2023-02-07 в 12 55 48
> 
> how much?

I dont remember, what we discuss last time?

**Comment by nastyakuzusha** [comment]
> > > Should it be 5 cards here? Which number we discussed previously? Снимок экрана 2023-02-07 в 12 54 43
> > > we didnt discuss numbers
> 
> we did Gun Part List Cards amount (2-3-4-5-6?)REVIEW Check with 5, make choice what is better 4 or 5 by yourself

check latest commit, its 5 here

**Comment by nastyakuzusha** [comment]
> > > Should it be smaller abount here? Снимок экрана 2023-02-07 в 12 55 48
> > 
> > 
> > how much?
> 
> I dont remember, what we discuss last time?

we didnt discuss this size of the screens, I can set it up to 8, as it is on XL screens now

**Pull request merged**